conf/modules/video_usb_logger.xml

```diff
@@ -12,7 +12,6 @@
     <define name="VIDEO_USB_LOGGER_HEIGHTH" value="272" description="Size of the to log images"/>
     <define name="VIDEO_USB_LOGGER_JPEG_WITH_EXIF_HEADER" value="TRUE" description="Whether to store data in the exif header or not"/>
   </doc>
-  <depends>video_rtp_stream</depends>
```

Comment:
jep, depends on video_thread, the needed jpeg.c should be added to the sources here directly...